libnopegl/src/ngpu/opengl/ctx_gl.c

```diff
@@ -494,13 +494,62 @@ static void ngpu_ctx_info_init(struct ngpu_ctx *s)
     s->limits = gl->limits;
 }
 
+static int create_command_buffers(struct ngpu_ctx *s)
+{
+    struct ngpu_ctx_gl *s_priv = (struct ngpu_ctx_gl *)s;
+
+    s_priv->update_cmd_buffers = ngli_calloc(s_priv->nb_in_flight_frames, sizeof(struct ngpu_cmd_buffer_gl *));
+    s_priv->draw_cmd_buffers = ngli_calloc(s_priv->nb_in_flight_frames, sizeof(struct ngpu_cmd_buffer_gl *));
+    if (!s_priv->update_cmd_buffers || !s_priv->draw_cmd_buffers)
+        return NGL_ERROR_MEMORY;
+
+    for (uint32_t i = 0; i < s_priv->nb_in_flight_frames; i++) {
+        s_priv->update_cmd_buffers[i] = ngpu_cmd_buffer_gl_create(s);
```

Comment:
Non-blocking: even if `nb_in_flight_frames` is `u32`, I think it makes sense to have the iterator in `size_t` to avoid the register extends for array accesses. Even `i32` would be better here since the undefined behavior allow the compiler to make the optimization, which is not the case with `u32`.

Reply:
I prefer to keep it that way for now as we already do that in the Vulkan backend.